.gitignore

```diff
@@ -37,4 +37,8 @@ nosetests.xml
 .pydevproject
```

Comment:
Should be also these patters written as 
*. pydevproject
*.project
*.mr.developer.cfg